index.html

```diff
@@ -49,5 +49,169 @@ <h1 class="h1">I'm Ezechukwu moses Glad to see you!</h1>
+        </div>
+      </div>
+
+      <!-- second work section -->
+
+      <div class="card">
+        <img class="img" src="./image/Snapshoot2.png" alt="ID" />
+        <div class="card-contents"></div>
+        <h2>Mult-Post Stories</h2>
+        <ul class="works">
+          <li>Canopy</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul >
+          <ul class="works2">
+          <li class="work2-li"><img src="./image/Tag.png" alt="html" /></li>
+          <li><img src="./image/Tag1.png" alt="css" /></li>
+          <li><img src="./image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <div class="div">
+          <button class="btn-big btn">see project</button>
+        </div>
+      </div>
+      
+
+      <!-- third work section -->
+
+      <div class="card">
+        <img class="img" src="./image/Snapshoot3.png" alt="ID" />
+        <div class="card-contents"></div>
+        <h2>Tonic</h2>
+        <ul class="works">
+          <li>CANOPY</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="works2">
+          <li><img src="./image/Tag.png" alt="html" /></li>
+          <li><img src="./image/Tag1.png" alt="css" /></li>
+          <li><img src="./image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <div class="div">
+          <button class="btn-big btn">see project</button>
+        </div>
+      </div>
+
+      <!-- fourth work section -->
+
+      <div class="card">
+        <img class="img" src="./image/Snapshoot4.png" alt="ID" />
+        <div class="card-contents"></div>
+        <h2>Mult-Post Stories</h2>
+        <ul class="works">
+          <li>CANOPY</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="works2">
+          <li><img src="./image/Tag.png" alt="html" /></li>
+          <li><img src="./image/Tag1.png" alt="css" /></li>
+          <li><img src="./image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <div class="div">
+          <button class="btn-big btn">see project</button>
+        </div>
```

Comment:
- [ ] [OPTIONAL] As my colleague suggested reducing `font-weight` and other things in your about me section below, I think you should try to make some adjustments to your properties on your project cards too - so that your design looks even more impressive 💯 

- **Your design**
![image](https://user-images.githubusercontent.com/99715640/212444051-036deaea-d8b0-492c-801c-68632f321350.png)

- [ ] [OPTIONAL] If you check the hover state of your button, you will find that the `box-shadow` needs some adjustment. Please check figma design for correct properties to correctly display the states of the button i.e. `hover`, `active`, `enabled` etc; I have attached an image of the pallet on Figma:
![image](https://user-images.githubusercontent.com/99715640/212444199-9f6b5191-fa4c-44ab-91cf-2a4eb5a52df9.png)

This will greatly reflect your rare aesthetic intuitions! 🛠️ 💪

Reply:
Thanks for the review @AbayomiOlaoye